config.sub

```diff
@@ -247,6 +247,7 @@ case $basic_machine in
 	# Some are omitted here because they have special meanings below.
 	1750a | 580 \
 	| a29k \
+	| aarch64 | aarch64_be \
```

Comment:
Two questions:

1. Why didn't we need this for iOS?
2. Can we add an `# airtime: CAR-601` comment here?

Reply:
1) Looks like for ios we are just setting the host string to `arm-apple-darwin`, then force the arch in subsequent xcrun: https://github.com/yoinc/deps/blob/8e367440c715c0c938b0f29ad10c9903fb1ffce3/third_party/libsrtp/run_config.sh#L64

2) Will do.

Reply:
Unfortunately I don't think it's possible to add the comment right above / below the line as it will break the string.. Should I put it in the comment block above?

Reply:
Yeah, put a comment after the first comment, and make a note that you added `aarch64` and `aarch64_be`.

Reply:
Done.